net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -555,8 +656,8 @@ function write_quagga_running_config($module) {
 	conf_mount_ro();
 }
 
-write_quagga_running_config('bgpd');
-write_quagga_running_config('ospfd');
-write_quagga_running_config('ospf6d');
-write_quagga_running_config('zebra');
+write_quagga_running_config(bgpd);
+write_quagga_running_config(ospfd);
+write_quagga_running_config(ospf6d);
+write_quagga_running_config(zebra);
```

Comment:
Why did you remove quotes here?

Reply:
Very good question. My production versions haven't had the quotes, so they must've been added in after I submitted the original code 18 months ago that included the "write_quagga_running_config()" function. I'll do some testing on my end to ensure it doesn't impact functionality.